ukrainian/story/eat.md

```diff
@@ -0,0 +1,5 @@
+﻿Ваш живіт вже переповнений зефіром і раптом вам захотілось
+[Зателефонувати своєму приятелеві](phone/phone.md)
+[Зробити отвір в кімнаті за допомогою ножа](knife/knife.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.